examples/howto/model_builder.myst.md

```diff
@@ -0,0 +1,292 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: pymc5
+  language: python
+  name: pymc5
+---
+
+# Using ModelBuilder class for deploying PyMC models 
+:::{post} Sep 12, 2022
+:tags: deployment
+:category: Advanced
+:author: Shashank Kirtania, Thomas Wiecki
+:::
+
++++
+
+## Motivation
+
++++
+
+Many users face difficulty in deploying their PyMC models to production because deploying/saving/loading a user-created model is not well standardized. One of the reasons behind this is there is no direct way to save or load a model in PyMC like scikit-learn or TensorFlow. The new `ModelBuilder` class is aimed to improve this workflow by providing a scikit-learn inspired API to wrap your PyMC models.
+
+The new `ModelBuilder` class allows users to use methods to `fit()`, `predict()`, `save()`, `load()`. Users can create any model they want, inherit the `ModelBuilder` class, and use predefined methods.
+
++++
+
+Let's go through the full workflow, starting with a simple linear regression PyMC model as it's usually written. Of course, this model is just a place-holder for your own model.
+
+```{code-cell} ipython3
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pymc as pm
+
+from numpy.random import RandomState
+
+%config InlineBackend.figure_format = 'retina'
+RANDOM_SEED = 8927
+
+rs = RandomState(RANDOM_SEED)
```

Comment:
is it necessary to use a randomstate specifically? If possible we should use [`Generator`](https://numpy.org/doc/stable/reference/random/generator.html) as randomstate (and the global state) are legacy code. 

From https://numpy.org/doc/stable/reference/random/legacy.html#legacy-random-generation:

> The [RandomState](https://numpy.org/doc/stable/reference/random/legacy.html#numpy.random.RandomState) provides access to legacy generators. This generator is considered frozen and will have no further improvements. [...] This class should only be used if it is essential to have randoms that are identical to what would have been produced by previous versions of NumPy.